Formula/brainfuck.rb

```diff
@@ -0,0 +1,30 @@
+class Brainfuck < Formula
+  desc "Interpreter for brainfuck written in C"
```

Comment:
not sure I understand the description since the formula itself is named `brainfuck`

Reply:
The name of this project, an interpreter for the brainfuck _language_, just "happens" to be named brainfuck as well, but they're essentially different entities (one is a language, and the other is an interpreter).  I'll see if I can come up with a less circulair appearing description however.